### ISSUE TITLE
Standalone examples can no longer be copied out and run standalone

### ISSUE THREAD

**Issue by ibgreen** [uber/deck.gl#2136]
As of 6.0 copy website example to a directory outside of deck.gl repo, install and run doesn't work due to dependency on HTML template.

#### Actual Result
- Standalone example fails to run

#### Expected Result
- Example runs

#### Reproduce Steps

```
cp -r examples/website/3d-heatmap /tmp
cd /tmp/3d-heatmap
yarn
yarn start
``` 

#### To Do List 
  - [ ] Add label and assign to milestone
  - [ ] Coding
  - [ ] Test

**Comment by Pessimistress** [comment]
We could just copy and paste the html file into each example.

**Comment by ibgreen** [comment]
> We could just copy and paste the html file into each example.

Yes... One of my PRs for last release was to remove all html from the examples. It wasn't (and shouldn't) be needed. 

If we are adding a file, could we not just have a two-line `root.js` or `index.js` in each example. 
```
import app from '/app';
render(..., App); // or some variation
```

(Above is assuming that the original solution of having a flag indicating that we are running under the demo environment is not desirable.)

**Comment by Pessimistress** [comment]
One effort during this release was to align the standalone examples behavior with the website - e.g. tooltips. The html template includes the necessary CSS.
